client/signup/steps/woocommerce-install/index.tsx

```diff
@@ -13,6 +12,7 @@ interface WooCommerceInstallProps {
 	goToStep: GoToStep;
 	stepName: string;
 	stepSectionName: string;
+	isReskinned: boolean;
```

Comment:
Is this temporary until we've completed all of the steps?

Reply:
Not sure what do you mean here. `isReskinned` in this case is used to apply some CSS class to the step wrapper element.